.azurePipeline/cibuildwheel_steps.yml

```diff
@@ -4,8 +4,12 @@ steps:
 - bash: |
     # Use this fork of cibuildwheel for macos c++11 support
     # https://github.com/joerick/cibuildwheel/pull/156
-    python -m pip install git+https://github.com/Czaki/cibuildwheel
+    # python -m pip install git+https://github.com/Czaki/cibuildwheel
```

Comment:
I guess we can now remove the comments above if we use directly the mypy package.

Reply:
Yeah Wilko and I will deal with MacOS and fix this up